### PR TITLE
[IMP] website: improve onboarding flow

### PR DESCRIPTION
**Pull request by ans-odoo**
- Improve the website tour (typo, tip positions, etc) and added new
  steps to publish a new page
- Make close button of 'Mobile Preview' more visible
- Fix the overflow for long menu names in menu editor dialog
- Change the menu string from 'Affix Top Menu' to 'Fixed Top Menu'
- Change theme customization dialog's title to 'Customize Theme'
  from 'Customize this theme'
- Improve strings for Google Analytics

task-1895287

### CODE REVIEW

File: addons/website/static/src/xml/website.editor.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <div t-name="website.homepage_editor_welcome_message" class="container text-center o_homepage_editor_welcome_message">
         <h2 class="mt0">Welcome to your <b>Homepage</b>!</h2>
         <p class="lead d-none d-md-block">Let's start designing.</p>
-        <div class="o_tooltip_container d-none d-md-block fade">Follow all the <div class="o_tooltip bottom"/> signs to get your website ready in no time.</div>
+        <div class="o_tooltip_container d-none d-md-inline-flex fade">Follow all the <div class="o_tooltip bottom"/> signs to get your website ready in no time.</div>
```
Comment (qsm-odoo) [inline addons/website/static/src/xml/website.editor.xml]: Also move this in the 12.0 PR (with the related scss rule)

File: addons/website/static/src/xml/website.contentMenu.xml [hunk 2]
```diff
@@ -6,8 +6,8 @@
             <div class="input-group-prepend">
                 <span class="input-group-text fa fa-bars" role="img" aria-label="Dropdown menu" title="Dropdown menu"/>
             </div>
-            <span class="form-control">
-                <span class="js_menu_label">
+            <span class="form-control d-flex">
+                <span class="js_menu_label o_text_overflow">
```
[the remark this answers is not in the document]
Comment (ans-odoo) [reply addons/website/static/src/xml/website.contentMenu.xml]: In menu editor, while home page (the one having fa-home symbol at the right) is having a long string, the home symbol does not look good. Screenshot : ( https://drive.google.com/file/d/1StqTmHYcbrX4vHhxcKs-iu33C16JM6P5/view ) You're right about floating elements though. Could you help me here with a proper fix?

[the remark this answers is not in the document]
Comment (qsm-odoo) [reply addons/website/static/src/xml/website.contentMenu.xml]: That home symbol do not appear anymore from 12.0, is this a bug?
You can use d-flex but would have to adapt the children element (maybe removing float-right and margin-top and adding "align-items-center" on the "d-flex" element ?)

[the remark this answers is not in the document]
Comment (ans-odoo) [reply addons/website/static/src/xml/website.contentMenu.xml]: removed margin-top, float-right and added align-self-center to the home symbol now it's aligned

[the remark this answers is not in the document]
Comment (ans-odoo) [reply addons/website/static/src/xml/website.contentMenu.xml]: I've made changes as you say. 
I think it's not a bug home symbol only appear when `is_homepage` field is set for that page.

[the remark this answers is not in the document]
Comment (qsm-odoo) [reply addons/website/static/src/xml/website.contentMenu.xml]: Standard 12.0, homepage is a homepage, no icon. Don't you have that too?

[the remark this answers is not in the document]
Comment (ans-odoo) [reply addons/website/static/src/xml/website.contentMenu.xml]: By default, there is no homepage icon on the homepage,
but when we set a new page as a homepage in that case homepage icon is displayed.
So we still need to fix
https://drive.google.com/open?id=1saWiSaP-ZOkSBYbEFdeW4KuwEgEBv2K6

**Pull request closed without merging**